lib/modules/manager/tekton/readme.md

```diff
@@ -13,16 +13,17 @@ Tasks and Pipeline definitions can also live outside the Kubernetes cluster and
 The `tekton` manager focuses on providing updates to Tekton resource references.
 
 Right now, Renovate's Tekton manager only supports references that are [Bundles](https://tekton.dev/docs/pipelines/tekton-bundle-contracts/).
-See the [`tektoncd/resolution` project on GitHub](https://github.com/tektoncd/resolution) for the different kinds of Tekton references.
+Read the [Tekton Pipeline remote resolution docs](https://tekton.dev/docs/pipelines/resolution/) for the different kinds of Tekton references and their corresponding resolvers.
 
 ### Using a Tekton Bundle reference
 
-There are two ways to use a Tekton Bundle reference:
+There are three ways to use a Tekton Bundle reference:
```

Comment:
is this still true? i only see two list items

Reply:
It should be:

1. Via the [Tekton Bundles Resolver](https://tekton.dev/docs/pipelines/bundle-resolver/)
1. Via the [`tektoncd/resolution` project](https://github.com/tektoncd/resolution)
1. Via the `taskRun.spec.taskRef.bundle` and the `pipelineRun.spec.pipelineRef.bundle` attributes

I believe that's what the diff says.

Reply:
i don't know. i only see that it needs a third list items or revert to two. 🤷‍♂️😉

Reply:
Maybe I'm missing something... Here's what the rendered markdown file looks like:
![image](https://user-images.githubusercontent.com/5272931/204647601-979d8118-d21d-488e-be88-eeb1d767abe9.png)